### PR TITLE
Issue #SB-13288 feat: added separate section for Upcoming Batches

### PR DESCRIPTION
**Pull request by raghav14**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
     "rxjs-compat": "^6.5.2",
     "sb-cordova-plugin-customtabs": "git+https://github.com/Sunbird-Ed/sb-cordova-plugin-customtabs.git",
     "sb-cordova-plugin-db": "git+https://github.com/swayangjit/sb-cordova-plugin-db.git",
-    "sunbird-sdk": "git+https://github.com/Sunbird-Ed/sunbird-mobile-sdk.git#v2.4.6-alpha",
+    "sunbird-sdk": "git+https://github.com/Sunbird-Ed/sunbird-mobile-sdk.git#v2.4.4-alpha",
```
Comment (swayangjit) [inline package.json]: Revert package.json changes.

File: src/app/components/cards/coursecard/coursecard.component.ts [hunk 2]
```diff
@@ -9,9 +9,11 @@ import {
 } from 'sunbird-sdk';
 import { InteractSubtype, InteractType, Environment, PageId } from '../../../../services/telemetry-constants';
 import { CommonUtilService } from '../../../../services/common-util.service';
-import { EnrollmentDetailsPage } from '@app/app/enrolled-course-details-page/enrollment-details-page/enrollment-details-page';
 import { Router } from '@angular/router';
+import forEach from 'lodash/forEach';
```
Comment (swayangjit) [inline src/app/components/cards/coursecard/coursecard.component.ts]: Don't use loadash foreach , use default for loop.

File: src/app/search/search.page.ts [hunk 3]
```diff
@@ -8,6 +8,7 @@ import { Location } from '@angular/common';
 import each from 'lodash/each';
 import find from 'lodash/find';
 import map from 'lodash/map';
+import forEach from 'lodash/forEach';
```
Comment (swayangjit) [inline src/app/search/search.page.ts]: Remove loadash.

File: src/app/components/cards/coursecard/coursecard.component.ts (hunk 2 above)
Comment (raghav14) [reply src/app/components/cards/coursecard/coursecard.component.ts]: Okay, will make that change

**Pull request merged**